src/screens/Books/components/ListItem/styles.js

```diff
@@ -0,0 +1,33 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  headerContent: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'space-around'
```

Comment:
Use `flex start` here and add some padding to the headerText

Reply:
Done on the next commit.